mongokit/apply.go

```diff
@@ -300,3 +301,16 @@ func applyCurrentDate(ctx Context, doc bsonkit.Doc, name, path string, v interfa
 
 	return nil
 }
+
+func applyPush(ctx Context, doc bsonkit.Doc, _, path string, v interface{}) error {
+	// add value
+	res, err := bsonkit.Push(doc, path, v)
+	if err != nil {
+		return err
+	}
+
+	// record change
+	ctx.Value.(*Changes).Updated[path] = res
+
```

Comment:
Since the $push operator supports advanced modifiers we need to leave a note about this: Please add a `// TODO: ...`  comment explaining which things are still missing.

Reply:
Done!